src/components/login/LoginForm.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const LoginForm = () => (
+  <form>
+    <div className="row mt-5 text-center">
+      <div className="col-md-8 offset-md-2">
+        <div className="input-group mb-3">
+          <div className="input-group-prepend">
+            <span className="input-group-text transparent email-addon" id="basic-addon1"><i className="far fa-envelope" /></span>
+          </div>
+          <input type="text" className="form-control email-input" placeholder="EMAIL ADDRESS" aria-label="Username" aria-describedby="basic-addon1" />
```

Comment:
Should this be a textType?

Reply:
it should be 'email'. Thanks for the feedback